### PR TITLE
Ibp 4367 releases notes

### PR DESCRIPTION
**Pull request by lik-83**
Pls, check documentation https://ibplatform.atlassian.net/wiki/spaces/DEV/pages/2057535563/Release+Notes for further information.

### CODE REVIEW

File: src/main/jhipster/src/main/webapp/app/release-notes/release-notes.component.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+@Component({
+    selector: 'jhi-release-notes-dialog',
+    templateUrl: './release-notes.component.html',
+    styleUrls: [
+        'release-notes.scss'
+    ],
+})
+export class ReleaseNotesDialogComponent implements OnInit {
+
+    dontShowAgain = true;
```
Comment (nahuel-soldevilla) [inline src/main/jhipster/src/main/webapp/app/release-notes/release-notes.component.ts:23]: dontShowAgain by default seem kind of weird. The user: "I like to see this popup next time, so I will uncheck this to make sure that happens"

File: src/main/java/org/generationcp/ibpworkbench/security/WorkbenchAuthenticationSuccessHandler.java [hunk 2]
```diff
@@ -49,10 +53,11 @@ public void onAuthenticationSuccess(final HttpServletRequest request, final Http
 		}
 
 		final WorkbenchUser user = retrieveUserFromAuthentication(authentication);
+		final Optional<ReleaseNote> releaseNote = this.releaseNoteService.shouldShowReleaseNote(user.getPerson().getId());
```
Comment (nahuel-soldevilla) [inline src/main/java/org/generationcp/ibpworkbench/security/WorkbenchAuthenticationSuccessHandler.java:56]: It seems to me optional is kind of unnecessary here, boolean, true or false feels more appropriate. It should be possible to map all possible states of the system to `shouldShowReleaseNote()` -> either true or false, even when there are scenarios where release note info might not exist (null).

**Review by nahuel-soldevilla — APPROVED** [review]
Looks great

File: src/main/jhipster/src/main/webapp/i18n/en/release-notes.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "release-notes": {
+        "title": "Upgrade Notification",
+        "description": "BMS has been upgraded since your last login. See more information below",
```
Comment (nahuel-soldevilla) [inline src/main/jhipster/src/main/webapp/i18n/en/release-notes.json:4]: If the user unchecks "dont show again", this description won't be accurate for the next time it sees the notification. Maybe simplify it to just _BMS has been upgraded **recently**. See more information below_

cc @mcrimi @clarysabel

File: src/main/java/org/generationcp/ibpworkbench/security/WorkbenchAuthenticationSuccessHandler.java (hunk 2 above)
Comment (lik-83) [reply src/main/java/org/generationcp/ibpworkbench/security/WorkbenchAuthenticationSuccessHandler.java]: Done!

File: src/main/jhipster/src/main/webapp/app/release-notes/release-notes.component.ts (hunk 1 above)
Comment (lik-83) [reply src/main/jhipster/src/main/webapp/app/release-notes/release-notes.component.ts]: Done!

File: src/main/jhipster/src/main/webapp/i18n/en/release-notes.json (hunk 3 above)
Comment (lik-83) [reply src/main/jhipster/src/main/webapp/i18n/en/release-notes.json]: Done!

File: src/main/jhipster/src/main/webapp/app/release-notes/release-notes.component.ts (hunk 1 above)
Comment (clarysabel) [reply src/main/jhipster/src/main/webapp/app/release-notes/release-notes.component.ts]: dontShowAgain will not be checked by default. Agreed with @mcrimi

File: src/main/jhipster/src/main/webapp/i18n/en/release-notes.json (hunk 3 above)
Comment (clarysabel) [reply src/main/jhipster/src/main/webapp/i18n/en/release-notes.json]: We will modify the description as suggested. Agreed with @mcrimi

**Pull request merged**